tests/samplers_tests/test_samplers.py

```diff
@@ -289,8 +298,7 @@ def test_intersection_search_space() -> None:
 
     # Failed or pruned trials are not considered in the calculation of
     # an intersection search space.
-    def objective(trial, exception):
-        # type: (optuna.trial.Trial, Exception) -> float
+    def objective(trial: optuna.trial.Trial, exception: Exception) -> float:
```

Comment:
Not introduced by this PR but we can simplify this line.
```suggestion
    def objective(trial: Trial, exception: Exception) -> float:
```

Reply:
Thx. Applied